kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -398,6 +410,18 @@ def _trim_job_tag(self, name):
         pattern = r"(-\d{4,10}$)"
         return re.sub(pattern, '', name)
 
+    def _extract_job_timestamp(self, name):
+        """
+        Extract timestamp of job names
+        """
+        ts = name.split('-')[-1]
+        if ts.isdigit():
+            return int(ts)
+        else:
+            msg = 'Cannot extract ts from job name {}'.format(name)
+            self.log.debug(msg)
+            return 0
```

Comment:
Just checking, is that the only way to get the job timestamp ?

At first glance, seems not optimal to rely on job name to extract the job timestamp.